From 2ce9635a7ea399e5370f4232409e7b17e40f6dfd Mon Sep 17 00:00:00 2001
From: Zeng Zhaoming <b32542@freescale.com>
Date: Mon, 4 Jul 2011 01:56:48 +0800
Subject: ENGR00152479 Early uart: Correct Copyright declaration

Early uart has more than one contributors.
Remove "all right reserve" declare in source code.

Signed-off-by: Zeng Zhaoming <b32542@freescale.com>
---
 drivers/tty/serial/mxc_uart_early.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/tty/serial/mxc_uart_early.c b/drivers/tty/serial/mxc_uart_early.c
index 46ca2b1..c970dc6 100644
--- a/drivers/tty/serial/mxc_uart_early.c
+++ b/drivers/tty/serial/mxc_uart_early.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2011 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright (C) 2011 Freescale Semiconductor, Inc.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
-- 
1.7.9.5

